src/client/src/orangehrmAttendancePlugin/pages/ViewEmployeeAttendanceSummary.vue

```diff
@@ -95,13 +95,16 @@ import {freshDate, formatDate} from '@ohrm/core/util/helper/datefns';
 import EmployeeAutocomplete from '@/core/components/inputs/EmployeeAutocomplete';
 import {yearRange} from '@/core/util/helper/year-range';
 import useDateFormat from '@/core/util/composable/useDateFormat';
+import usei18n from '@/core/util/composable/usei18n';
+
+const {$t} = usei18n();
 
 const attendanceRecordNormalizer = data => {
   return data.map(item => {
     return {
       id: item.empNumber,
       empName: `${item?.firstName} ${item?.lastName}
-          ${item?.terminationId ? ' (Past Employee)' : ''}`,
+          ${item?.terminationId ? ` ${$t('general.past_employee')}` : ''}`,
```

Comment:
Composable functions should be exclusively used within the setup context. 
https://vuejs.org/guide/reusability/composables.html#using-composables-in-options-api

```
<template>
  <div>test</div>
</template>

<script>
// outside of component context
// no access to any reactive variables/methods
// only used for imports and static functions

export default {
  name: 'SomeComponent',
  setup() {
    // the setup context
    // this is called first, hence it don't have access to whats defined in data()
    // all the composables are consumed here
    // dont have access to mixins or global attributes
    // unless explicitly defined
  },
  data() {
    // component context
    // has access to whats defined in setup
    // if returned from setup
  },
  // or computed/ other hooks
  methods: {
    // component context
    // has access to whats defined in setup
    // if returned from setup
  }
};
</script>
```